bindgen/spec.yml

```diff
@@ -570,6 +570,18 @@ records:
       fast_reconnect_limit:
         type: uint64_t
         default: 60000
+      resumption_delay_interval:
+        type: uint64_t
+        default: 1000
+      max_resumption_delay_interval:
+        type: uint64_t
+        default: 300000
+      resumption_delay_backoff_multiplier:
+        type: int
+        default: 2
+      resumption_delay_jitter_divisor:
+        type: uint8_t
```

Comment:
Why is this a uint8_t? It's just a plain int down in the sync client

Reply:
Removed uint8_t since it is no longer broken out to it's own parameter.